lib/diagnostic.rb

```diff
@@ -6,35 +6,42 @@ class Diagnostic
   # a Fixnum.
   def longest(list)
     # Your code goes here
+    list.max
   end
 
   # Question 2
   # Write ruby code to return the sum of entries in the
   # list supplied (i.e. apply the `+` operator).
-  # Assume each entry implements a `+` method.
+  # Assume each entry implements  a `+` method.
   def sum(list)
     # Your code goes here
+    sum = 0
+    list.each.to_i { |i| sum += i }
```

Comment:
Yes but as the instructions say, and what we learned, you can call `list.reduce(&:+)`